addons/odoo_payments/models/adyen_account.py

```diff
@@ -194,6 +210,74 @@ def _check_vat(self):
                     "The given registration number is invalid: %s",
                     account.registration_number))
 
+    @api.depends('company_id')  # fake 'depends' to be sure that this _compute method is called when the form view is displayed
+    def _compute_need_to_provide_payment_journal(self):
+        self.need_to_provide_payment_journal = self.env['ir.module.module'].sudo().search([
+            ('name', '=', 'account_accountant'),
+            ('state', '=', 'installed'),
+        ])
+
+    @api.depends('company_id')
+    def _compute_payment_journal_id(self):
+        for account in self:
+            acquirer = self.env['payment.acquirer'].search([
+                ('provider', '=', 'odoo'),
+                ('company_id', '=', account.company_id.id),
+            ], limit=1)
+            account.payment_journal_id = acquirer.journal_id
+
+    def _inverse_payment_journal_id(self):
+        for account in self:
+            acquirer = self.env['payment.acquirer'].search([
+                ('provider', '=', 'odoo'),
+                ('company_id', '=', account.company_id.id),
+            ], limit=1)
+            acquirer.journal_id = account.payment_journal_id
+
+    @api.depends('state', 'transactions_count', 'adyen_kyc_ids', 'shareholder_ids', 'bank_account_ids')
+    def _compute_onboarding_msg(self):
+        self.onboarding_msg = False
+        for account in self:
+            if account.state == 'pending' and account.id:
+                account.onboarding_msg = (
+                    "Our team will review your account. We will notify you, by email, as soon "
+                    "as you can start processing payments."
+                )
```

Comment:
Missing translation for all the onboarding message parts ;)
```suggestion
                account.onboarding_msg = _(
                    "Our team will review your account. We will notify you, by email, as soon "
                    "as you can start processing payments."
                )
```

Reply:
Oups, right :+1: